Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentEnd/FocusHelper.js

```diff
@@ -26,14 +36,149 @@ const handler = (event) => {
       });
     }
   }
+
+  if (eventType.toUpperCase() === "FOCUS") {
```

Comment:
I don't think `eventType` exists in this context

Reply:
Also, event names should always come in lowercase, so the transform to upper shouldn't be needed.